### ISSUE TITLE
Add width,height and viewBox attributes to SVG element

### ISSUE THREAD

**Issue by basilinjoe** [flying-sheep/SWF2SVG#2]
Add width, height, and viewBox attributes to the SVG element?

**Comment by berteh** [comment]
yes please. Or i have to use inkscape to correct each of them manually.

**Comment by flying-sheep** [comment]
Hi! I haven’t touched this for years, but I’ll accept PRs of course. Anyone feel up for the task?

**Comment by berteh** [comment]
not me sorry, never touched AIR in my life and no time to start now. would be nice though.

**Comment by flying-sheep** [comment]
It wasn’t super hard, I haven’t touched it before or since either :wink:
